### PR TITLE
feat: consolidate contributing documentation

### LINKED ISSUE #79: basic contributing docs
## acceptance criteria
* document install and setup process
* document tech stack
* document basic workflow

### PR DESCRIPTION
**Pull request by arrocke**
closes #79

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Our goal is to build tools that make it easy for groups of people to translate B
 
 We are still in the very early stages. At present we are working on a tool for producing interlinear Bibles in any language. From there we have rough plans for taking what we have learned and applying it to tools for translating lexicons, and other original language tools.
 
+In the GitHub project, you can see the [high level state of the project](https://github.com/users/arrocke/projects/1/views/5) as well as [what tasks are pioritized and in progress](https://github.com/users/arrocke/projects/1/views/1).
```
Comment (Pertempto) [inline README.md]:
```suggestion
In the GitHub project, you can see the [high level state of the project](https://github.com/users/arrocke/projects/1/views/5) as well as [what tasks are prioritized and in progress](https://github.com/users/arrocke/projects/1/views/1).
```

File: docs/contributing.md [hunk 2]
```diff
@@ -2,15 +2,40 @@
...
 
-## Getting Started
+**Build tools:** npm, nx
 
-You can set up your development environment by following [these steps](./setup.md).
+**Web client:** react, tailwind
+
+**API server:** node, nextjs, prisma, postgresql
+
+## Local Development
```
Comment (Pertempto) [inline docs/contributing.md:15]: It would be good to have a note about recommended terminals in each OS. Is that part of a future issue?

**Review by Pertempto — CHANGES_REQUESTED** [review]

File: docs/contributing.md (hunk 2 above)
Comment (arrocke) [reply docs/contributing.md:15]: I don't think the terminal should matter too much. It should work for sure in bash (including git bash) and zsh. I'm not sure what you are using. I added a note to that effect.

Comment (Pertempto) [reply docs/contributing.md:15]: PowerShell wasn't working well at all, so I switched to Git bash and it works much better.

**Review by Pertempto — APPROVED** [review]

**Pull request merged**